src/Aspire.Hosting.AWS/README.md

```diff
@@ -157,6 +157,107 @@ builder.AddProject<Projects.Frontend>("Frontend")
        .WithReference(construct, c => c.Url, "Url");
 ```
 
+## Integrating AWS Lambda Local Development
+
+You can develop and test AWS Lambda functions locally within your .NET Aspire application. This enables testing Lambda functions alongside other application resources during development.
+
+![\[\]!(.)](Resources/lambda.png)
+
+
+### Adding Lambda Functions
+
+To add a Lambda function to your .NET Aspire AppHost, use the `AddAWSLambdaFunction` method. The method supports both executable Lambda functions and class library Lambda functions:
+
+```csharp
+#pragma warning disable CA2252 // This API requires opting into preview features
+
+// Add an executable Lambda function
+builder.AddAWSLambdaFunction<Projects.ExecutableLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ExecutableLambdaFunction")
+    .WithReference(awsConfig);
+
+// Add a class library Lambda function
+builder.AddAWSLambdaFunction<Projects.ClassLibraryLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ClassLibraryLambdaFunction::ClassLibraryLambdaFunction.Function::FunctionHandler")
+    .WithReference(awsConfig);
+
+```
+
+The handler parameter specifies the Lambda handler in different formats depending on the project type:
+
+- For executable projects: specify the assembly name.
+- For class library projects: use the format `{assembly}::{type}::{method}`.
```

Comment:
Can you add a sub section about how the Amazon.Lambda.TestTool is a .NET tool that is automatically installed by the integration. The installation can be configured using the `AddAWSLambdaServiceEmulator` method.